pkg/test/handler/utils.go

```diff
@@ -0,0 +1,30 @@
+package handler
```

Comment:
I agree that `handlertest` is redundant. My thinking was these two packages go together and `handlertest` gives immediate context to what the package is for (testing handler). And since we're using an alias anyway, it felt to me like there was an unnecessary package name collision.

But we can leave this for now, maybe I'll think of a different name change later.